scripts/dev/freeze.py

```diff
@@ -65,6 +65,7 @@ def get_build_exe_options(skip_html=False):
         ('qutebrowser/git-commit-id', 'git-commit-id'),
         ('qutebrowser/utils/testfile', 'utils/testfile'),
         ('qutebrowser/html', 'html'),
+        ('qutebrowser/3rdparty/pdfjs', '3rdparty/pdfjs'),
```

Comment:
They now run it, but when you give no or invalid parameters, it will update pdfjs and then fail and print an error message. I don't know how I should fix this, as argument parsing and validation is done withing `cx.setup` - I'm already calling `update_3rdparty` as late as possible, calling it after `cx.setup` would kind of defeat its purpose.